golden_hour/location.py

```diff
@@ -12,7 +14,7 @@
             'latitude': schema.And(schema.Use(float), lambda lat: -90 < lat < 90),
             'longitude': schema.And(schema.Use(float), lambda lon: -180 < lon < 180),
             'timezone': str,
-            'elevation': schema.And(schema.Use(float), lambda elevation: elevation >= 0),
+            'elevation': schema.And(schema.Use(float), lambda elevation: elevation >= -LOWEST_ALLOWED_ELEVATION),
```

Comment:
This is a double-negative, right? So the minimum elevation is `-(-100)`?

Reply:
oops

Reply:
fixed